src/app/core/models/iteam.ts

```diff
@@ -0,0 +1,8 @@
+export interface ITeam{
```

Comment:
Is ITeam a typo for Team? Maybe NBATeam would be more specific, unless the other free soccer API we plan on using has the same exact "team" data model, then Team would work.

Reply:
Not a typo just the naming convention we use in C# and what I have seen in Angular for interfaces.